packages/openapi-generator/src/sdk-metadata/generation-and-usage.ts

```diff
@@ -62,14 +60,14 @@ function getApiSpecificUsage(
       openApiDocument.serviceOptions.packageName
     );
     return {
-      instructions,
-      text: apiSpecificUsageText,
+      ...instructions,
+
       header: usageHeaderText
     };
   }
   return {
     instructions: '',
-    text: apiSpecificUsageText,
+    text: '',
```

Comment:
minor: I see many cases that we set empty string to `text`, which means this can be optional. The same to other properties like `instructions`.

Reply:
we have an issue for handling also the cases where there is no text yet.